it/api/SetupDesResponseISpec.scala

```diff
@@ -30,7 +30,7 @@ import scala.concurrent.ExecutionContext
 
 class SetupDesResponseISpec extends IntegrationSpecBase with MongoIntegrationSpec with APIHelper {
 
-  implicit val ex: ExecutionContext = implicitly[Application].actorSystem.dispatcher.prepare()
+  implicit val ex: ExecutionContext = implicitly[Application].actorSystem.dispatcher
```

Comment:
Can this be replaced by this `import scala.concurrent.ExecutionContext.Implicits.global`?